test/blackbox-tests/test-cases/glob_files_rec.t

```diff
@@ -9,7 +9,7 @@ be release as it. We plan to replace it by recursive globs for 3.0.0.
   > (rule
   >  (alias x)
   >  (deps (glob_files_rec foo/*.txt))
-  >  (action (bash "for i in %{deps}; do printf \"\$i\\n\"; done")))
+  >  (action (bash "for i in %{deps}; do printf \"\$i\\n\"; done | sort")))
```

Comment:
I'm surprised this is necessary. If dune doesn't return `%{deps}` in some deterministic order, then it would certainly be a bug in dune.

Reply:
Ah good point. I've updated `Glob_files.expand_*` to sort their results internally.